### PR TITLE
Update custom image to datahub doc

### PR DESCRIPTION
**Pull request by aakankshaduggal**
## Related Issues and Dependencies
https://github.com/orgs/aicoe-aiops/projects/5#card-40148435
…

## This Pull Request implements

After moving all our repositories to github, the reference links have changed.

### CODE REVIEW

**Comment by goern** [comment]
/assign aakankshaduggal
/unassign harshad16

@aakankshaduggal you can fix these problems yourself on your host, try to `pre-commit install` and `pre-commit run --all-files` on the repo. The key is `- files were modified by this hook` pre-commit cant fix on a PR, just the use can do it on his clone.

File: aicoe/add-image-to-datahub.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Import Container Image to Internal Datahub:
 
-1. Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+* Create a new ImageStream template, you can use https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub[this template].
```
Comment (harshad16) [inline aicoe/add-image-to-datahub.md]: why is the link referencing format changed?

Comment (aakankshaduggal) [reply aicoe/add-image-to-datahub.md]: My bad @harshad16, fixed that now!

**Comment by harshad16** [comment]
> Pre-Commit Test failed! Click here

@aakankshaduggal this end-of-file-fixer issue is occurring due to the formatting.

File: aicoe/add-image-to-datahub.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 # Import Container Image to Internal Datahub:
 
-1. Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+* Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
+* Fork this [repository](https://github.com/AICoE/aicoe-sre/tree/master/applications/jupyterhub/bases).
+* Add a new ImageStream to the [repository](https://github.com/AICoE/aicoe-sre/blob/master/applications/jupyterhub/bases/custom-images).
```
Comment (4n4nd) [inline aicoe/add-image-to-datahub.md]: the link should be https://github.com/AICoE/aicoe-sre/blob/master/applications/jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

File: aicoe/add-image-to-datahub.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
 # Import Container Image to Internal Datahub:
 
-1. Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+* Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
+* Fork this [repository](https://github.com/AICoE/aicoe-sre/tree/master/applications/jupyterhub/bases).
```
Comment (4n4nd) [inline aicoe/add-image-to-datahub.md]: this link should be https://github.com/AICoE/aicoe-sre

File: aicoe/add-image-to-datahub.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
+* Add a new ImageStream to the [repository](https://github.com/AICoE/aicoe-sre/blob/master/applications/jupyterhub/bases/custom-images).
+* Create a Pull Request to publish these changes to the internal DataHub instance.
 
-The newly created Merge Request will be then reviewed and tested by the DataHub team, and when after all the testing the merge request is merged, your notebook image should be available in the internal Datahub/Jupyterhub instance.
+The newly created Pull Request will be then reviewed and tested by the DataHub team, and when after all the testing the merge request is merged, your notebook image should be available in the internal Datahub/Jupyterhub instance.
+
+For further reference, have a look at one of our previous [Merge Requests](https://github.com/AICoE/aicoe-sre/pull/25) to see what the process looks like.
```
Comment (4n4nd) [inline aicoe/add-image-to-datahub.md]: Can you change Merge Request to Pull Request like Tom suggested?

**Review by 4n4nd — CHANGES_REQUESTED** [review]
Requested a few changes. Can you also rebase?

File: aicoe/add-image-to-datahub.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 # Import Container Image to Internal Datahub:
+- Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
```
Comment (harshad16) [inline aicoe/add-image-to-datahub.md:3]: Leave a space after title.

File: aicoe/add-image-to-datahub.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+The newly created Pull Request will be then reviewed and tested by the DataHub team, and when after all the testing the merge request is merged, your notebook image should be available in the internal Datahub/Jupyterhub instance.
 
-The newly created Merge Request will be then reviewed and tested by the DataHub team, and when after all the testing the merge request is merged, your notebook image should be available in the internal Datahub/Jupyterhub instance.
+For further reference, have a look at one of our previous [Merge Requests](https://github.com/AICoE/aicoe-sre/pull/25) to see what the process looks like.
 
-You can look at one of our previous [Merge Requests](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/merge_requests/27) to see what the process looks like.
```
Comment (harshad16) [inline aicoe/add-image-to-datahub.md:10]: leave only one newline at the end.

**Review by harshad16 — COMMENTED** [review]
Lgtm :+1: 
minor comment to fix lint issue.

**Review by harshad16 — APPROVED** [review]
Thanks :medal_military:

**Pull request merged**